abi_test.go

```diff
@@ -1,6 +1,7 @@
 package ebpf
 
 import (
+	"github.com/cilium/ebpf/internal/testutils"
```

Comment:
NIT: This doesn't match the order / grouping of the rest of the imports

Reply:
Fixed, thanks.